addons/l10n_ar/models/account_fiscal_position.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models, api
+
+
+class AccountFiscalPosition(models.Model):
+
+    _inherit = 'account.fiscal.position'
+
+    l10n_ar_afip_responsability_type_codes = fields.Char(
+        'AFIP Responsability Type Codes',
+        help='List of AFIP responsability codes where this fiscal position '
+        'should be auto-detected',
+    )
+
+    def get_fiscal_position(self, partner_id, delivery_id=None):
+        """ Send this afip_responsability of the partner to _get_fpos_by_region
+        """
+        company = self.env['res.company'].browse(self._context.get(
+            'force_company', self.env.user.company_id.id))
+        if company.country_id == self.env.ref('base.ar'):
+            partner = self.env['res.partner'].browse(partner_id)
+            afip_responsability = \
+                partner.commercial_partner_id.l10n_ar_afip_responsability_type
+            self = self.with_context(
+                partner_afip_responsability=afip_responsability)
```

Comment:
No way to put the logic below in _get_fpos_by_region here?

Reply:
the logic is on _get_fpos_by_region the problem is that the method "_get_fpos_by_region" doesn't receive the partner, so we send what we need on the context.

Reply:
we are trying to put the login on get_fiscal_position